app/views/services/govbox/register_step2.html.erb

```diff
@@ -7,27 +7,79 @@
         <h1 class="form-title">Údaje štatutára</h1>
         <p class="form-subtitle lead">Zadajte údaje štatutára (napríklad konateľa s.r.o.), ktorý podpíše splnomocnenie na prístup k schránke.</p>
 
-        <div class="row">
-          <div class="col-md-4 form-group">
-            <%= label_tag :given_name, 'Meno' %>
-            <%= text_field_tag :given_name, params[:given_name], class: 'form-control input-lg', required: true %>
+        <% if @statutory_entries %>
+          <h6>Vyberte možnosť:</h6>
+          <% @statutory_entries.each_with_index do |person, index| %>
+            <div class="row my-3">
+              <div class="col-md-12">
+                <label class="form-check-label">
+                  <%= radio_button_tag :statutory_entry, index, false, class: 'form-check-input', required: true, data: { given_name: person[:first_name], family_name: person[:last_name], person_formatted_address: format_address(person) } %>
```

Comment:
vyhodme ten helper

plus, `format_address` je velmi genericky nazov pre takto specificky helper (vzhladom na to co vsetko tento projekt obsahuje)

```suggestion
                  <%= radio_button_tag :statutory_entry, index, false, class: 'form-check-input', required: true, data: { given_name: person[:first_name], family_name: person[:last_name], person_formatted_address: Datahub::Utils.build_formatted_address(OpenStruct.new(person)) } %>
```